src/utils/utils.py

```diff
@@ -158,12 +161,26 @@ def unescape_moses(moses_tokens):
     return [_MOSES_DETOKENIZER.unescape_xml(t) for t in moses_tokens]
 
 
-def process_sentence(sent, max_seq_len, sos_tok=SOS_TOK, eos_tok=EOS_TOK):
+def process_sentence(sent, max_seq_len, tokenizer='', start_tok=SOS_TOK, end_tok=EOS_TOK):
```

Comment:
We should probably make the tokenizer argument required, no? Otherwise there's room for subtle bugs.

Reply:
:+1: